tests/Fixtures/filters/nl2br.test

```diff
@@ -3,6 +3,7 @@
 --TEMPLATE--
 {{ "I like Twig.\nYou will like it too.\n\nEverybody like it!"|nl2br }}
 {{ text|nl2br }}
+{{ null|nl2br }}
```

Comment:
```suggestion
*{{ null|nl2br }}*
```

So that we have `**` in the output (like I did in the referenced PR)

Reply:
Can you also add a test with an empty string? (again, for consistency with the referenced PR)

Reply:
Done